tests/conftest.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+from rdkit import Chem
+
+from chemprop.data.datapoints import MoleculeDatapoint
+
+_DATA_DIR = Path(__file__).parent / "data"
+_DF = pd.read_csv(_DATA_DIR / "smis.csv")
+_DF["mol"] = _DF["smiles"].map(Chem.MolFromSmiles)
+_DF["smi"] = _DF["mol"].map(Chem.MolToSmiles)
+
+
+@pytest.fixture
+def data_dir():
+    return _DATA_DIR
+
+
+@pytest.fixture
+def smis():
+    return _DF.smi
+
+
+@pytest.fixture
+def mols():
+    return _DF.mol
+
+
+@pytest.fixture
+def targets(smis):
+    return np.random.rand(len(smis), 1)
+
+
+@pytest.fixture
+def mol_data(mols, targets):
+    return [MoleculeDatapoint(mol, y) for mol, y in zip(mols, targets)]
+
+
+@pytest.fixture
+def rxn_data(rxns, targets):
+    return [MoleculeDatapoint(mol, y) for mol, y in zip(mols, targets)]
+
+
+@pytest.fixture(params=_DF.smi.sample(5))
+def smi(request):
+    return request.param
+
+
+@pytest.fixture(params=_DF.mol.sample(5))
+def mol(request):
+    return request.param
+
+
+@pytest.fixture
+def mol_regression_data(data_dir):
```

Comment:
This fixture isn't used. `test/integration/test_regression_mol.py` uses `mol_data`, while `test/integration/test_regression_rxn.py` uses `rxn_regression_data`. Is there a reason these two integration tests aren't symmetric in that way?

Reply:
I've changed it now and commented the `{mol,rxn}_data` fixtures out for now. Perhaps there will be a use for them in the future.